test/e2e/framework/metrics_util.go

```diff
@@ -52,9 +52,11 @@ const (
 	apiCallLatencyThreshold time.Duration = 1 * time.Second
 
 	// We use a higher threshold for list apicalls if the cluster is big (i.e having > 500 nodes)
-	// as list response sizes are bigger in general for big clusters.
-	apiListCallLatencyThreshold  time.Duration = 5 * time.Second
-	bigClusterNodeCountThreshold               = 500
+	// as list response sizes are bigger in general for big clusters. We also use a higher threshold
+	// for list calls with cluster scope (all namespaces).
+	apiListCallLatencyThreshold      time.Duration = 5 * time.Second
+	apiClusterScopeListCallThreshold time.Duration = 10 * time.Second
```

Comment:
Do you literally mean cluster scoped (i.e., node objects) or cross-namespace lists? If the latter, please choose a less confusing name.

Reply:
I meant the former (which includes both non-namespaced and all-namespaced calls) - changed the comment to make it clearer.